torch/onnx/verification.py

```diff
@@ -199,6 +199,15 @@ def _compare_ort_pytorch_outputs(
                         f"within acceptable range {acceptable_error_percentage}."
                     )
                     continue
+            else:
+                if (
+                    ort_out.dtype == torch.uint8 or ort_out.dtype == torch.int8
+                   ) == True:
```

Comment:
No need for == True. I would remove `else:` and dedent the new code as well. ort_out and pt_out are all np arrays so we should compare dtype to np types.

Otherwise looking good

Reply:
Remove else:, ==True and changed dtype check from torch to np